test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/listModels.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var util = require('util');
+var async = require('async');
+var app = require('../../../../../');
+var loopback = require('loopback');
+var path = require('path');
+var ModelConfig = app.models.ModelConfig;
+var exampleWorkspace = path.resolve(__dirname, '../../../../../example');
+
+module.exports = function() {
+  var numberOfExpectedModels = 0;
+  var numberOfAvailableModels = 0;
+  var testcase = this;
+
+  this.Given(/^I have a workspace containing (\d+) models$/, function(numberOfExpectedModels, next) {
+    testcase.numberOfExpectedModels = parseInt(numberOfExpectedModels);
```

Comment:
I also think this part is convoluted. You can probably get rid of the testcase stuff by renaming the variables:

```
var expectedModelCount;

this.Given('I have a workspace containing "$count" models', function(count, next) {
  expectedModelCount = count;
  next();
});
```